packages/core/src/resolveTrustChains/policies/errors/PolicyMergeError.ts

```diff
@@ -0,0 +1,17 @@
+import type { MetadataPolicyOperator } from '../../../metadata'
+
+type Details = {
+  path: string
+  operatorA: MetadataPolicyOperator
+  operatorB: MetadataPolicyOperator
+}
+
+export class PolicyOperatorMergeError extends Error {
```

Comment:
class is named `PolicyOperatorMergeError`, but name is set to `PolicyMergeError`. Shouldn't these be equal?